.yamlfmt

```diff
@@ -0,0 +1,7 @@
+doublestar: true
```

Comment:
Without doublestar, the exclude didn't work.
Looks different issue of https://github.com/google/yamlfmt/issues/97